utility/src/main/java/org/oppia/util/parser/SvgSoftwareLayerSetter.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.util.parser
+
+import android.graphics.drawable.PictureDrawable
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.target.Target
+
+/**
+ * Listener which updates the [ImageView] to be software rendered, because {@link
```

Comment:
Per my earlier comment thread, I still don't understand why this actually works (e.g. how returning false in both methods below causes it to be software rendered). Do you have additional documentation for this?

Reply:
Actual implementation of SvgSoftwareLayerSetter is [here](https://gist.github.com/veena14cs/42ad32de4d3b4c0fdb7eb0d6c7c46617). But I am getting class cast exception and also we are not loading in Imageview we are loading in our custom target, so I removed part of the implementation of Imageview which was taking software rendering.

Now I tried to render without adding listener to the glide. And after removing this svg image is getting loaded, it is making no difference.

Reply:
Ah, thanks for clarifying. If this ends up being an issue (e.g. we switch one of our layers to use hardware) it looks like we can fix it in our own code by changing the software layer.